code/modules/uplink/uplink_items.dm

```diff
@@ -511,6 +511,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "An alternative 8-round Scatter Laser Shot magazine for use in the Bulldog shotgun."
 	item = /obj/item/ammo_box/magazine/m12g/scatter
 	cost = 5 // most armor has less laser protection then bullet
+	include_modes = list(/datum/game_mode/nuclear)
+
+/datum/uplink_item/ammo/shotgun/chemdarts
+	name = "12g Dart Shotgun Shells"
+	desc = "Two 8-round dart magazine for use in the Bulldog shotgun. \
+            Great for mixing up a plethora of death. Dosn't come with a toxin kit, needs to be filled by hand..."
+	item = /obj/item/storage/box/syndie_kit/chemdarts
+	include_modes = list(/datum/game_mode/nuclear)
```

Comment:
The base cost of 2Tc is way too cheap here, especially when the toxins kits are going to cost 6 or less if they're surplus (Probably the only time they'll get used, honestly.)

The chemical kit comes with 13 bottles of 30 units of toxins. That's going to equate to 11 highly likely kills, 3 empty shells and a big balance issue.

So, assuming 50% accuracy, that's 6.7 kills on targets, hardsuits notwithstanding, for 8 TC. At 100% accuracy, it could claim top lethality on the syndicate loadout.

By comparison you have the bioterror darts, which inject the same amount of a far less dangerous chem (and are not available for purchase via uplink), and bioterror grenades and sprays which can be protected against via internals and chemical suits. For the darts, there's no counter beyond a hardsuit.

For reference, Bioterror costs 6 for 7 SYNRINGES (105 units, aka 3.5 darts).

So this is completely hecked up. Purchasing the poison kit, at base cost, would be broken and having it on surplus would be completely bananas at 5TC for a 13 shot murder gun. As it stands, syndicate standard slugs do not one-shot people (42 damage ea. if target is naked), but all but two of the drugs in the Toxins kit will give a target enough of a damage type to down them, ensuring or simply giving a rapid kill in the case of Cyanide. Basically a long range tazer with shotgun accuracy deviation, hence the generous 50% accuracy rate.

I think the only exceptions I found were Spewium, which guarentees delayed stuns anyway, and Histamine which does pretty sizeable damage over time anyway.

In short, Bioterror shotgun darts aren't in the uplink and these are better than them, so I don't think these should be included in the Syndie kit either.

Reply:
As I investigated this, I found the best combination was to mix 1 unit of every chemical in the toxins kit into one dart, because it was a guarenteed down.